### PR TITLE
models.stats: Add clt_version and variants fields

### PR DESCRIPTION
**Pull request by arjunsalyan**

### CODE REVIEW

**Review by mojca — APPROVED** [review]

**Comment by arjunsalyan** [comment]
This PR is probably complete now.

File: app/ports/migrations/0006_add_clt_version_and_variants.py
Comment (mojca) [inline app/ports/migrations/0006_add_clt_version_and_variants.py:16]: I cannot really decide how much we need here, and would be tempted to either use perhaps 256 or just text with no limits. But if we use text, maybe the performance could decrease. Maybe 200 is sufficient?

File: app/ports/models/stats.py
Comment (mojca) [inline app/ports/models/stats.py]: Didn't we say that we wouldn't put any defaults here?

File: app/ports/migrations/0007_alter_field_clt_version.py
Comment (mojca) [inline app/ports/migrations/0007_alter_field_clt_version.py:16]: Do we need two migrations in the same PR? (Maybe we do because of the deployed version.)

**Review by mojca — APPROVED** [review]

File: app/ports/migrations/0007_alter_field_clt_version.py
Comment (arjunsalyan) [reply app/ports/migrations/0007_alter_field_clt_version.py:16]: Yes, the deployed demo version needs that we put these migration files.

File: app/ports/models/stats.py
Comment (arjunsalyan) [reply app/ports/models/stats.py]: Oh, thank you very much. It is from a previous commit and got overlooked.

File: app/ports/migrations/0007_alter_field_clt_version.py
Comment (arjunsalyan) [reply app/ports/migrations/0007_alter_field_clt_version.py:16]: We now have three. But there is a way to avoid this, we can replace them with a single file and then run a fake migration.

Comment (mojca) [reply app/ports/migrations/0007_alter_field_clt_version.py:16]: I leave that up to you. I'm fine with any solution, so feel free to merge it yourself in the proper way.

**Review by mojca — COMMENTED** [review]
As I already said last time, I'm ok with the changes, so feel free to press the merge button.

There's one missing piece that could be changed. Running a query that fetches all old submissions which lack `clt_version` or `variants` field, but contain `raw_json` field, and fill in the missing information.

That's somewhat optional and can be done in a separate PR to avoid postponing this one. But we might need something similar later as well, so it could be useful to set an example now. On the deployed version you can no longer manually change the database.

**Pull request merged**
